src/controllers/IBackendController.ts

```diff
@@ -0,0 +1,20 @@
+import { Express } from "express";
+import { ParamsDictionary, RequestHandler } from "express-serve-static-core";
+
+/*
+ * Standard interface for Backend controllers
```

Comment:
As we are in the context of a backend app, I think it safe to avoid specifying "backend" here